cloud/role/role.go

```diff
@@ -0,0 +1,85 @@
+package role
+
+import (
+	httpContext "context"
+	"io"
+
+	astrocore "github.com/astronomer/astro-cli/astro-client-core"
+	"github.com/astronomer/astro-cli/context"
+	"github.com/astronomer/astro-cli/pkg/printutil"
+)
+
+var rolePagnationLimit = 100
+
+// Returns a list of all of an organizations roles
+func GetOrgRoles(client astrocore.CoreClient) ([]astrocore.Role, []astrocore.DefaultRole, error) {
+	offset := 0
+	var roles []astrocore.Role
+
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return nil, nil, err
+	}
+	var defaultRoles []astrocore.DefaultRole
+	for {
+		includeDefaultRoles := true
+		resp, err := client.ListRolesWithResponse(httpContext.Background(), ctx.Organization, &astrocore.ListRolesParams{
+			IncludeDefaultRoles: &includeDefaultRoles,
+			Offset:              &offset,
+			Limit:               &rolePagnationLimit,
+		})
+		if err != nil {
+			return nil, nil, err
+		}
+		err = astrocore.NormalizeAPIError(resp.HTTPResponse, resp.Body)
+		if err != nil {
+			return nil, nil, err
+		}
+		defaultRoles = *resp.JSON200.DefaultRoles
```

Comment:
nit can we do it so we only set this once and after the first `ListRolesWithResponse` call, we turn `includeDefaultRoles to false` since we already have em

Reply:
actually should we add a flag as to whether to include it or not,
seems kinda annoying to have it always show up if you only care about custom roles
im thinking `--include-default-roles` should even be default false 🤔

Reply:
good call. done.